articles/search/cognitive-search-skill-entity-recognition.md

```diff
@@ -49,7 +49,7 @@ Parameters are case-sensitive and are all optional.
 ## Skill outputs
 
 > [!NOTE]
-> Not all entity categories are supported for all languages. Only _en_, _es_ support extraction of `"Quantity"`, `"Datetime"`, `"URL"`, `"Email"` types.
+> Not all entity categories are supported for all languages. The `"Person"`, `"Location"`, and `"Organization"` entity category types are supported for the full list of languages above. Only _de_, _en_, _es_, _fr_, and _zh-hans_ support extraction of `"Quantity"`, `"Datetime"`, `"URL"`, and `"Email"` types. For more information, see https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/language-support.  
```

Comment:
When you copy and paste links into documentation, make sure to remove the "en-us" from the url so that it will be localized for whoever clicks on it instead of defaulting to the US/English page.

Reply:
Thanks @Careyjmac!  Done in commit a4c4ab044fd7099bcc7d174d8101069ddcbd58c8.